sdk/test-utils/test-utils/src/tracing/azureTracing.ts

```diff
@@ -0,0 +1,127 @@
+import { OperationTracingOptions, useInstrumenter } from "@azure/core-tracing";
+import { assert } from "chai";
+import { TestInstrumenter } from "./testInstrumenter";
+import { SpanGraph, SpanGraphNode } from "./spanGraphModel";
+// this is the plugin used in the test file
+function chaiAzureTrace(chai: Chai.ChaiStatic, _utils: Chai.ChaiUtils): void {
```

Comment:
Few things I wonder about:
1. Should we add some JSDoc comments to provide some information on what this adds?
2. Should we rename this to `chaiAzure`?
3. Doesn't this need to be exported from index?

Reply:
1. I will add more JSDoc comments
2. I am ok with either. We can also wait for the rename when we decide to extend the functionality to other than tracing?
3. This is exported from the index. It should show